WordPress/src/main/res/layout/login_include_epilogue_header.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <FrameLayout
+        android:id="@+id/login_epilogue_header_avatar_layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:background="?attr/selectableItemBackgroundBorderless"
+        android:clickable="false"
+        android:contentDescription="@string/content_description_add_avatar"
+        android:enabled="false"
+        android:focusable="true">
+
+        <ImageView
+            android:id="@+id/login_epilogue_header_avatar"
+            android:layout_width="@dimen/avatar_sz_login_epilogue"
+            android:layout_height="@dimen/avatar_sz_login_epilogue"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_user_circle_no_padding_grey_24dp" />
+
+        <ImageView
+            android:id="@+id/login_epilogue_header_avatar_add"
+            android:layout_width="@dimen/add_avatar_button_size"
+            android:layout_height="@dimen/add_avatar_button_size"
+            android:layout_gravity="top|end"
+            android:background="@drawable/bg_oval_surface"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_add_white_24dp"
+            android:visibility="gone"
+            app:tint="?attr/colorPrimary"
```

Comment:
NIT: I think we no longer need the `?attr/`, `?colorPrimary` should be enough

Reply:
You're right. I'm only keeping the `?attr/` because that's the format we've been using the most throughout the app.